src/util/fetchUtils.js

```diff
@@ -27,6 +41,12 @@ export const parseQueryResponse = queries => ({ responses, error, message }) =>
  * @return {Promise} resolves with a `{queries, responses}` object
  */
 export const fetchQueries = (apiUrl, options) => (queries, meta) => {
+	if (
+		typeof window === 'undefined' &&  // not in browser
+		typeof test === 'undefined'  // not in testing env
```

Comment:
yes. Not very happy about having to reference test-specific vars in production code, but I've been starting to think that we should make a clear separation between node code and browser code in the platform, and if we do that, then we can make sure to run tests on this in a simulated browser environment and this can be stripped out again.

I'll make a note to bring it up in our next weekly.